documentation/docs/community/contribute/tips-and-tricks.md

```diff
@@ -0,0 +1,27 @@
+# Tips and tricks
+
+This section provides useful snippets and how-tos while developping software for the PlanktoScope.
+
+!!! warning
+
+    This document is meant for PlanktoScope developers. Proceed with care.
+
+## Backup and Restore SD Card
+
+You will need to plug the SD card into your computer.
+
+`/dev/device` refers to the path of the SD card device/disk. You will need to adjust it. Use `diskutil list` on macOS and `fdisk --list` on Linux.
+
+
+```sh
+# backup whole SD card onto an image file on your computer
+sudo dd bs=1M if=/dev/device status=progress | xz > sdcard.img.xz
+```
+
+```sh
+# restore an image file from your computer onto the SD card
+xzcat sdcard.img.xz | sudo dd bs=1M of=/dev/mmcblk0 status=progress
```

Comment:
I wonder if it might be worth also linking to https://docs-edge.planktoscope.community/setup/software/nonstandard-install/#write-the-os-image-to-an-sd-card and/or replicating your `xzcat sdcard.img.xz | sudo dd bs=1M of=/dev/mmcblk0 status=progress` command to that section.

Reply:
We used to build OS images as `.img.gz`, but I later changed it to `.img.xz` to get more consistent compression so that they fit within the 2 GB  size limit for GitHub Release attachments. Anything which refers to `.img.gz` is something I forgot to update to `.img.xz`.

Reply:
Thanks!